libethcore/Farm.h

```diff
@@ -48,6 +48,16 @@ namespace dev
 {
 namespace eth
 {
+struct FarmSettingsType
+{
+    unsigned DagLoadMode = 0;  // 0 = Parallel; 1 = Serialized
```

Comment:
I'd prefer to keep the existing naming convention and start fields with lower case.

Reply:
Ok

Reply:
Casing corrected.